includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -61,7 +61,6 @@ private function getLangWithFallback( $lang ) {
 			'global-sitemap' => 'http://www.wikia.com/Sitemap',
 			'local-sitemap' => '/wiki/Local_Sitemap',
 			'local-sitemap-fandom' => 'http://fandom.wikia.com/local-sitemap',
-			'api' => 'http://api.wikia.com/wiki/Wikia_API_Wiki',
 			'community-central' => 'http://community.wikia.com/wiki/Community_Central',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "community-central" 15 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)